app/lib/db.js

```diff
@@ -0,0 +1,24 @@
+const redis = require('redis');
+const { promisify } = require('util');
+const config = require('../models/config-model').server;
+
+const client = redis.createClient(
+    config.redis.cache.port,
+    config.redis.cache.host,
+    {
+        auth_pass: config.redis.cache.password,
+    }
```

Comment:
As soon as I saw "LGTM" show up in my inbox, I took a second look and realized that this isn't complete. Obviously this only addresses one of two Redis instances. My inclination is to export two clients rather than one, which I think will have the added benefit of making this distinction clearer going forward.